src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -59,14 +60,16 @@ public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone
         this.birthdate = birthdate;
         this.race = race;
         this.religion = religion;
-        this.survey = survey;
+        if (survey != null) {
+            this.survey.addAll(survey);
+        }
         if (tagged != null) {
             this.tagged.addAll(tagged);
         }
     }
 
     /**
-     * Converts a given {@code Person} into this class for Jackson use.
+     * Converts a given {@code PJsonAdaptedTagerson} into this class for Jackson use.
```

Comment:
Not sure what this is supposed to mean. Probably a typo?

Reply:
Thanks for pointing it out. Will fix them now.

Reply:
Fixed typo.